### PR TITLE
[benchmark] Skip broken benchmark on 8 cores.

### PR DESCRIPTION
**Pull request by paulosjca**
This change skips one specific benchmark on 8 cores that only passes on 30 cores. The benchmark hangs and does not complete when run on the 8-core node pool. The 30-core node pool has `c2-standard-30` nodes, compared to `e2-standard-8` for the 8-core node pool.

The skipped benchmark is the following:

- `cpp_protobuf_async_client_unary_1channel_64wide_128breq_8mbresp_insecure`

With this change, all benchmarks on both node pools should be passing.

### CODE REVIEW

**Review by veblush — APPROVED** [review]
Thanks for the fix.

One question; Is this considered as a temporary workaround to make the test pass while trying to fix it (e.g. enabling the timed-out test again on 8 core pool) or making this change sort of permanent?

File: tools/internal_ci/linux/grpc_e2e_performance_gke.sh [hunk 1]
```diff
@@ -190,8 +209,18 @@ if [[ -v "useLanguage[ruby]" ]]; then
   runnerLangArgs+=(-l "ruby:${GRPC_CORE_REPO}:${GRPC_CORE_COMMIT}")
 fi
 
-buildConfigs "${WORKER_POOL_8CORE}" "${BIGQUERY_TABLE_8CORE}" "${configLangArgs8core[@]}"
-buildConfigs "${WORKER_POOL_32CORE}" "${BIGQUERY_TABLE_32CORE}" "${configLangArgs32core[@]}"
+# Disable broken tests by regex.
+declare -a disabledTests8core=(
+  cpp_protobuf_async_client_unary_1channel_64wide_128breq_8mbresp_insecure
+)
```
Comment (veblush) [inline tools/internal_ci/linux/grpc_e2e_performance_gke.sh:215]: Would you add comments why this needs to be disabled?

File: tools/internal_ci/linux/grpc_e2e_performance_gke_experiment.sh [hunk 2]
```diff
@@ -185,8 +204,18 @@ if [[ -v "useLanguage[ruby]" ]]; then
   runnerLangArgs+=(-l "ruby:${GRPC_CORE_REPO}:${GRPC_CORE_COMMIT}")
 fi
 
-buildConfigs "${WORKER_POOL_8CORE}" "${BIGQUERY_TABLE_8CORE}" "${configLangArgs8core[@]}"
-buildConfigs "${WORKER_POOL_32CORE}" "${BIGQUERY_TABLE_32CORE}" "${configLangArgs32core[@]}"
+# Disable broken tests by regex.
+declare -a disabledTests8core=(
+  cpp_protobuf_async_client_unary_1channel_64wide_128breq_8mbresp_insecure
```
Comment (veblush) [inline tools/internal_ci/linux/grpc_e2e_performance_gke_experiment.sh:209]: Would you add comments why this needs to be disabled?

**Comment by paulosjca** [comment]
> Thanks for the fix.
> 
> One question; Is this considered as a temporary workaround to make the test pass while trying to fix it (e.g. enabling the timed-out test again on 8 core pool) or making this change sort of permanent?

It is a permanent fix. Made that clear in the comments.

**Review by veblush — APPROVED** [review]

**Pull request closed without merging**